ext/symengine/symengine.c

```diff
@@ -46,4 +47,10 @@ void Init_symengine() {
     c_rational = rb_define_class_under(m_symengine, "Rational", c_basic);
     rb_define_alloc_func(c_rational, cbasic_alloc);
     rb_define_method(c_rational, "initialize", crational_init, 1);
+
+    //Constants
+    c_constants = rb_define_class_under(m_symengine, "Constants", c_basic);
```

Comment:
@rajithv, can you add the statement in the above comment as a test?
Also something like `(x+E-x)instance_of? SymEngine::Constant`